src/bubble.py

```diff
@@ -0,0 +1,35 @@
+"""Implementation of Bubble sort in Python."""
+
+
+def bubble_sort(iter):
+    """Sort the iterable using the Bubble sort method."""
+    if not isinstance(iter, (list, tuple)):
+        raise TypeError("Input only a list/tuple of integers")
+    if not all(isinstance(x, (int, float)) for x in iter):
+        raise ValueError("Input only a list/tuple of integers")
+
+    size = 0
+    while size < (len(iter) - 1):
```

Comment:
You should avoid using `iter` as a variable name as that's a reserved word in pYthon